rancher/concepts/health-checks/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Health Checks with Rancher
+layout: rancher-default
+
+---
+
+#### Health Checks
+
+Rancher implements a distributed health monitoring system by running an HAProxy instance on every host for the sole purpose of providing health checks to containers.  When health checks are enabled either on an individual container or a service, each container is then monitored by up to three HAProxy instances running on hosts seperate to that containers parent host. The container is considered healthy if at least one HAProxy instance reports a "passed" health check.
+
+> **Note:** The only exception to this model is when your environment contains a single host, in this instance the health checks will be performed by the same host.
+
+Rancher’s approach handles network partitions and is more efficient than client-based health checks. By using HAProxy to perform health checks, Rancher enables users to specify the same health check policy across applications and load balancers.
+
+### Failure Scenarios
+
+Scenario | Response
+----|----
+* The container being monitored stops responding to health checks. | All active HAProxy instances monitoring the container would detect the failure and mark the container as "unhealthy". If the container is part of a service then rancher will restore the service to it's pre-defined scale through it's service HA functionality, on other hosts.
```

Comment:
"on other hosts" - not necessarily so. The container can be re-created on the very same host where it failed initially, as long as the host is considered up and available for containers allocation.